src/components/list/list.tsx

```diff
@@ -99,6 +99,12 @@ export class List {
     @Event()
     protected select: EventEmitter<ListItem | ListItem[]>;
 
+    /**
+     * Fires when a user interacts with an item in the list (e.g., click,
+     * keyboard select).
+     */
+    @Event() interaction: EventEmitter<ListItem>;
```

Comment:
We have a similar named event in a few other places, so for consistency we should rename this
```suggestion
    @Event() interact: EventEmitter<ListItem>;
```

Reply:
Also you should add the same event to `limel-menu-list`!

Reply:
Thanks Johan, I'll change the naming so it's more consistent throughout the package.

Reply:
See https://github.com/Lundalogik/lime-elements/pull/3332/commits/6a992ca2b9495958b83ba23d82a1fee84e3e4674